doc/RELEASE_WALKTHROUGH.rst

```diff
@@ -94,10 +94,12 @@ needed information.
 Finish the release notes
 ------------------------
 
-If this is the first release in a series the release note is generated, see
-the release note in ``doc/release/upcoming_changes/README.rst`` to see how to
-do this. Generating the release notes will also delete all the news
-fragment files in ``doc/release/upcoming_changes/``.
+If there are any release notes snippets in ``doc/release/upcoming_changes/``,
+run ``spin docs`` to build the docs, incorporate the contents of the generated
+``doc/source/release/notes-towncrier.rst`` file into the release notes file
+(e.g., ``doc/source/release/2.3.4-notes.rst``), and delete the now-processed
+snippets in ``doc/release/upcoming_changes/``. This is safe to do multiple
+times during a release cycle.
```

Comment:
I can't suggest changes below line 105, so I am commenting here: do the last lines of this paragraph need updating?